tests/secret_links/test_sharing.py

```diff
@@ -189,10 +184,10 @@ def test_permission_levels(service, restricted_record, identity, client):
 
 
 def test_read_restricted_record_with_secret_link(
-    app, client, location, minimal_record, es_clear, identity_simple
+    running_app, client, minimal_record, es_clear, identity_simple
 ):
     """Test access to a restricted record via a shared link."""
-    identity_simple.provides.add(authenticated_user)
+    # identity_simple.provides.add(authenticated_user)
```

Comment:
minor: `identity_simple` is an already authenticated user, so there is no need to add it. Can remove comment